Panic with lighthouse 0.0.922 when I comment /help

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x7995b4]

goroutine 49 [running]:
github.com/jenkins-x/lighthouse/pkg/scmprovider.(*Client).CreateComment(0xc0000e17a0, 0xc000665370, 0xa, 0xc00066535e, 0x2, 0x2b, 0xc00054e500, 0xc00014e000, 0x30d, 0x30d, ...)
	/workspace/source/pkg/scmprovider/issues.go:254 +0x594
github.com/jenkins-x/lighthouse/pkg/plugins/help.handle(0xc0002ab900, 0xc000665351, 0x4, 0x1d4ec00, 0xc0000e17a0, 0xc0001664d0, 0x1d04f40, 0xc000166700, 0xc00067f858, 0x672b21e9ead60898, ...)
	/workspace/source/pkg/plugins/help/help.go:150 +0x91f
github.com/jenkins-x/lighthouse/pkg/plugins/help.glob..func1(0x0, 0x0, 0xc000665351, 0x4, 0x0, 0x0, 0xc0000e17a0, 0x1d05020, 0xc000657280, 0x1d445a0, ...)
	/workspace/source/pkg/plugins/help/help.go:67 +0xbf
github.com/jenkins-x/lighthouse/pkg/webhook.(*Server).handleGenericComment.func2.1(0xc0005f7d10, 0xc00049a400, 0xc00037a280, 0xc000605298, 0x4, 0x1bbf5d8, 0x0, 0x0, 0xc000665351, 0x4, ...)
	/workspace/source/pkg/webhook/events.go:155 +0x163
created by github.com/jenkins-x/lighthouse/pkg/webhook.(*Server).handleGenericComment.func2
	/workspace/source/pkg/webhook/events.go:153 +0xf7
```

--- a/pkg/scmprovider/issues.go
+++ b/pkg/scmprovider/issues.go
@@ -239,23 +239,13 @@ func (c *Client) CreateComment(owner, repo string, number int, pr bool, comment 
 	if pr {
 		_, response, err := c.client.PullRequests.CreateComment(ctx, fullName, number, &commentInput)
 		if err != nil {
-			var b bytes.Buffer
-			_, cperr := io.Copy(&b, response.Body)
-			if cperr != nil {
-				return errors.Wrapf(cperr, "response: %s", b.String())
-			}
-			return errors.Wrapf(err, "response: %s", b.String())
+			return connectErrorHandle(response, err)
 		}
 
 	} else {
 		_, response, err := c.client.Issues.CreateComment(ctx, fullName, number, &commentInput)
 		if err != nil {
-			var b bytes.Buffer
-			_, cperr := io.Copy(&b, response.Body)
-			if cperr != nil {
-				return errors.Wrapf(cperr, "reponse: %s", b.String())
-			}
-			return errors.Wrapf(err, "response: %s", b.String())
+			return connectErrorHandle(response, err)
 		}
 	}
 	return nil
@@ -271,26 +261,28 @@ func (c *Client) EditComment(owner, repo string, number int, id int, comment str
 	if pr {
 		_, response, err := c.client.PullRequests.EditComment(ctx, fullName, number, id, &commentInput)
 		if err != nil {
-			var b bytes.Buffer
-			_, cperr := io.Copy(&b, response.Body)
-			if cperr != nil {
-				return errors.Wrapf(cperr, "response: %s", b.String())
-			}
-			return errors.Wrapf(err, "response: %s", b.String())
+			return connectErrorHandle(response, err)
 		}
 
 	} else {
 		_, response, err := c.client.Issues.EditComment(ctx, fullName, number, id, &commentInput)
 		if err != nil {
-			var b bytes.Buffer
-			_, cperr := io.Copy(&b, response.Body)
-			if cperr != nil {
-				return errors.Wrapf(cperr, "reponse: %s", b.String())
-			}
-			return errors.Wrapf(err, "response: %s", b.String())
+			return connectErrorHandle(response, err)
 		}
 	}
 	return nil
+}
+
+func connectErrorHandle(response *scm.Response, err error) error {
+	if response.Body != nil {
+		var b bytes.Buffer
+		_, cpErr := io.Copy(&b, response.Body)
+		if cpErr != nil {
+			return errors.Wrapf(cpErr, "response: %s", b.String())
+		}
+		return errors.Wrapf(err, "response: %s", b.String())
+	}
+	return errors.Wrapf(err, "no response body, code: %d", response.Status)
 }
 
 // ReopenIssue reopen an issue

